### DISCUSSION TITLE
Vercel Wont Pull Repo After Downgrading Account

### DISCUSSION THREAD

**Discussion by Xyliaze** [vercel/community#6406] [Help]
### Summary

I recently downgraded from pro because I simply have to need for t. (I'm using up 1% of bandwidth and nothing of anything else).

But after downgrading, I'm unable to push deployments.
They will push to GitHub, but vercel will no longer pull them from GitHub and build them.

If I upgrade again, and push the same code, vercel will pull that commit and build it just fine.

### Steps to Reproduce

- Upgrade account
- Push commit to repo
- Allow vercel to pull and build that commit

- Downgrade account
- Try to push same code to repo
- Vercel will fail to pull it

**Comment by amyegan** [comment]
Thanks for reaching out, @Xyliaze. If you disconnect the repo and then reconnect it, does that solve the problem?
